packages/enrichPrescriptions/tests/data/simple/requestBundleCancelled.json

```diff
@@ -0,0 +1,192 @@
+  "entry": [
+    {
+      "fullUrl": "urn:uuid:a67af4c7-da7c-4edd-a887-0c75a1b2963a",
+      "search": {
+        "mode": "match"
+      },
+      "resource": {
+        "resourceType": "Bundle",
+        "type": "collection",
+        "entry": [
+          {
+            "fullUrl": "urn:uuid:526b31df-e9ae-4423-b205-9692ec617469",
+            "resource": {
+              "id": "526b31df-e9ae-4423-b205-9692ec617469",
+              "identifier": [
+                {
+                  "system": "https://fhir.nhs.uk/Id/prescription-order-item-number",
+                  "value": "e76812cf-c893-42ff-ab02-b19ea1fa11b4"
+                }
+              ],
+              "courseOfTherapyType": {
+                "coding": [
+                  {
+                    "code": "acute",
+                    "display": "Short course (acute) therapy",
+                    "system": "https://terminology.hl7.org/CodeSystem/medicationrequest-course-of-therapy"
+                  }
+                ]
+              },
+              "dispenseRequest": {
+                "performer": {
+                  "reference": "urn:uuid:473e6278-af56-4767-9019-9757f4c0f595"
+                },
+                "quantity": {
+                  "code": "258682000",
+                  "system": "https://snomed.info/sct",
+                  "unit": "gram",
+                  "value": 15
+                },
+                "validityPeriod": {
+                  "start": "2023-11-12"
+                }
+              },
+              "groupIdentifier": {
+                "system": "https://fhir.nhs.uk/Id/prescription-order-number",
+                "value": "727066-A83008-2EFE36"
+              },
+              "intent": "order",
+              "medicationCodeableConcept": {
+                "coding": [
+                  {
+                    "code": "41898711000001103",
+                    "display": "Fusidic acid 2% cream",
+                    "system": "https://snomed.info/sct"
+                  }
+                ]
+              },
+              "requester": {
+                "reference": "urn:uuid:9d4afe92-348e-4f10-9ba5-67e7712eb8a1"
+              },
+              "status": "active",
+              "subject": {
+                "identifier": {
+                  "system": "https://fhir.nhs.uk/Id/nhs-number",
+                  "value": "5623367550"
+                }
+              },
+              "substitution": {
+                "allowedBoolean": false
+              },
+              "resourceType": "MedicationRequest",
+              "extension": [
+                {
+                  "url": "https://fhir.nhs.uk/StructureDefinition/Extension-DM-PrescriptionStatusHistory",
+                  "extension": [
+                    {
+                      "url": "status",
+                      "valueCoding": {
+                        "system": "https://fhir.nhs.uk/CodeSystem/task-businessStatus-nppt",
+                        "code": "Cancelled"
```

Comment:
I don't think we need new files for Cancelled and Prescriber Approved. You could make a function in tests/utils.ts that will add an extension to a medication request. Similar to the `defaultExtension()` function.

Reply:
The function `addExtensionToMedicationRequest` has been added to tests/utils.ts